### PR TITLE
Cybersixgill alerts typosquatting

### PR DESCRIPTION
**Pull request by syed-loginsoft**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added 2 incident fields to Incident type Cybersixgill Actionable Alert. Added a new mapper to map those to incident context data

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "classifier-mapper-incoming-Cybersixgill-Actionable-Alerts",
+    "mapping": {
+        "Cybersixgill Actionable Alerts": {
+            "dontMapEventToLabels": true,
+            "internalMapping": {
+                "Suspicious domain": {
+                    "simple": "additional_info.tables.[0].suspicious_domain"
```
Comment (melamedbn) [inline Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:10]: Please change to additional_info.tables.suspicious_domain if possible

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                },
+                "Triggered domain": {
+                    "simple": "additional_info.tables.[0].triggered_domain"
```
Comment (melamedbn) [inline Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:13]: Please change to additional_info.tables.triggered_domain

**Comment by melamedbn** [comment]
Hi @syed-loginsoft,

Great work! I added two comments directly to the Classifier file.
Besides that, please add a clearer description of each incident type so the user would better understand the triggered and suspicious domains' incident types.

Best regards,
Ben

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json (hunk 1 above)
Comment (kgal-pan) [reply Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:10]: @syed-loginsoft - I thought we discussed the _Suspicious domain_ will be a list of domains.

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json (hunk 2 above)
Comment (kgal-pan) [reply Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:13]: @syed-loginsoft - I thought we discussed the _Triggered domain_ will be a list of domains.

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_suspiciousdomain",
+    "version": -1,
+    "modified": "2023-11-09T22:08:56.499540505+05:30",
+    "name": "Suspicious domain",
+    "ownerOnly": false,
+    "description": "Suspicious Domain",
+    "cliName": "suspiciousdomain",
+    "type": "shortText",
```
Comment (kgal-pan) [inline Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json:9]: @syed-loginsoft - I thought we discussed the _Suspicious domain_ will be a list.

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_triggereddomain",
+    "version": -1,
+    "modified": "2023-11-09T22:08:00.46683194+05:30",
+    "name": "Triggered domain",
+    "ownerOnly": false,
+    "description": "Triggered domain",
+    "cliName": "triggereddomain",
+    "type": "shortText",
```
Comment (kgal-pan) [inline Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json:9]: @syed-loginsoft - I thought we discussed the _Triggered domain_ will be a list of domains.

**Comment by kgal-pan** [comment]
@syed-loginsoft - Since we're adding an incoming mapper, we can also add [these fields](https://github.com/demisto/content/blob/master/Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py#L135-L137) to the mapper and remove them from the Python code.

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json (hunk 3 above)
Comment (melamedbn) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json:9]: @kgal-pan Should @syed-loginsoft change the type to 'list'?

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json (hunk 4 above)
Comment (melamedbn) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json:9]: @kgal-pan Should @syed-loginsoft change the type to 'list'?

Comment (kgal-pan) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json:9]: @melamedbn - According to what we discussed on our call last week, the field should be a dynamic list.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py [hunk 5]
```diff
@@ -130,11 +130,6 @@ def get_alert_content(content_item, item_info, incident, sixgill_alerts_client):
         content_item['content'] = f'https://portal.cybersixgill.com/#/cve/{cve_id}'
         additional_info = item_info.get("additional_info", {})
         incident['CustomFields']['cve'] = cve_id
-        cybersixgillcvss31 = additional_info.get("nvd", {}).get("v3", {}).get("current")
```
Comment (sapirshuker) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py:133]: I think this change may break BC. @kgal-pan ?

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts_test.py [hunk 6]
```diff
@@ -215,9 +212,6 @@
     "CustomFields": {
         "cve": "Sample ID",
         "cybersixgillattributes": "",
-        "cybersixgillcvss20": -1,
```
Comment (sapirshuker) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts_test.py:218]: This case your validation to failed on test. please fix it.

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json (hunk 3 above)
Comment (kgal-pan) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json:9]: https://github.com/demisto/content/pull/30787#discussion_r1398367581

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml [hunk 7]
```diff
@@ -109,7 +109,7 @@ script:
       name: alert_status
     - description: The aggregate alert id.
       name: aggregate_alert_id
-  dockerimage: demisto/sixgill:1.0.0.79377
+  dockerimage: demisto/sixgill:1.0.0.80180
```
Comment (kgal-pan) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml:112]: Set the default incoming mapper for the integration:

```suggestion
  dockerimage: demisto/sixgill:1.0.0.80180
  defaultmapperin: classifier-mapper-incoming-Cybersixgill-Actionable-Alerts
```

Might be better to change the `id`, `name` properties of the classifier to something friendlier (e.g. _Cybersixgill Actionable Alerts Incoming Mapper_ and update the `defaultmapperin` here if you do.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py (hunk 5 above)
Comment (kgal-pan) [reply Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py:133]: Not sure this is change causes a BC. From what I see, the call stack is as follows:

```
get_alert_content > add_sub_alerts_fields > item_to_incidents > fetch_incidents
```

This method (`get_alert_content`) is used only during a fetch and these fields' mapping were moved to the incoming mapper which is used during the fetch and [was requested to be set up as the default incoming mapper](https://github.com/demisto/content/pull/30787#discussion_r1398459300). I think that it shouldn't affect users in this case unless they don't set up the incoming mapper.

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_suspiciousdomain",
```
Comment (sapirshuker) [inline Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json:2]: Please add a prefix to the file name incidenttype-Cybersixgill_Suspicious_Domain.json

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json [hunk 9]
```diff
@@ -0,0 +1,31 @@
+{
+	"description": "",
+	"feed": false,
+	"id": "classifier-mapper-incoming-Cybersixgill-Actionable-Alerts",
```
Comment (sapirshuker) [inline Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:4]: Might be better to change the `id`, `name` properties of the classifier to something friendlier (e.g. Cybersixgill Actionable Alerts Incoming Mapper) and update the defaultmapper in the yml file here if you do.

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Good work :)
Thank you for your contribution!
There are just a few small changes.
Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json [hunk 10]
```diff
@@ -0,0 +1,31 @@
+{
+	"description": "",
```
[the remark this answers is not in the document]
Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json]: Done

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json [hunk 11]
```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_triggereddomain",
```
[the remark this answers is not in the document]
Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Triggered_Domain.json]: Done

File: Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json (hunk 8 above)
Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json]: Done

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json (hunk 9 above)
Comment (syed-loginsoft) [reply Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json]: Done

**Comment by melamedbn** [comment]
Hi @syed-loginsoft,

Thanks for your work on the Cybersixgill pack. I've gone through it and have a few points that could use some tweaking:

### Incoming Mapper:
- **Naming Convention**: Let’s adjust the name to 'Cybersixgill Actionable Alerts - Incoming Mapper' for consistency with our naming standards. (missing hyphen) 
  
- **Field Mapping Accuracy**: Could you double-check the mappings of 'additional_info.nvd.v3.current' to 'Cybersixgill CVSS 2.0' and 'additional_info.nvd.v2.current' to 'Cybersixgill CVSS 3.1'? Just want to confirm that we've got these mapped correctly.

### Incident Fields:
- **Prefix Addition**: To keep with our best practice, please add the 'Cybersixgill' prefix to the relevant incident fields.

If you need any clarification on these points or have any questions, feel free to reach out.

Best regards,
Ben

**Comment by melamedbn** [comment]
> Hi @syed-loginsoft,
> 
> Thanks for your work on the Cybersixgill pack. I've gone through it and have a few points that could use some tweaking:
> 
> ### Incoming Mapper:
> * **Naming Convention**: Let’s adjust the name to 'Cybersixgill Actionable Alerts - Incoming Mapper' for consistency with our naming standards. (missing hyphen)
> * **Field Mapping Accuracy**: Could you double-check the mappings of 'additional_info.nvd.v3.current' to 'Cybersixgill CVSS 2.0' and 'additional_info.nvd.v2.current' to 'Cybersixgill CVSS 3.1'? Just want to confirm that we've got these mapped correctly.
> 
> ### Incident Fields:
> * **Prefix Addition**: To keep with our best practice, please add the 'Cybersixgill' prefix to the relevant incident fields.
> 
> If you need any clarification on these points or have any questions, feel free to reach out.
> 
> Best regards, Ben

The changes should be made also to the file name, and id and name keys of each field.

Best regards,
Ben

**Review by sapirshuker — COMMENTED** [review]
Hey @syed-loginsoft 
Thank you for your contribution 
Please see Ben's comment about the file name and the id of the fields.
In addition, you have some validation [errors](https://app.circleci.com/pipelines/github/demisto/content/320355/workflows/7a718f4a-4a9d-4f54-9c68-81b84a22acf7/jobs/730638).
Please feel free to reach out to me with any questions - I'm available here or on slack.
Thanks again

**Review by melamedbn — APPROVED** [review]

**Review by sapirshuker — APPROVED** [review]
Great Work!

**Pull request merged**